Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -677,56 +663,60 @@ def getSourcePaths(self, makeGlyphs=True, makeKerning=True, makeInfo=True):
             paths.append(self.documentObject.sources[name][0].path)
         return paths
 
-    def newDefaultLocation(self):
-        loc = {}
-        for axisDescriptor in self.axes:
-            loc[axisDescriptor.name] = axisDescriptor.default
-        return loc
-
     def readRules(self):
-        # read the rules
+        # we also need to read any conditions that are outside of a condition set.
         rules = []
         for ruleElement in self.root.findall(".rules/rule"):
             ruleObject = self.ruleDescriptorClass()
             ruleObject.name = ruleElement.attrib.get("name")
+            # read any stray conditions outside a condition set
+            externalConditions = self._readConditionElements(ruleElement)
+            if externalConditions:
+                ruleObject.conditionSets.append(externalConditions)
+                warnings.warn('Found stray rule conditions outside a conditionset. Wrapped them in a new conditionset.')
```

Comment:
instead of `warnings.warn` (which is more intended for orther developers using a library from their applications), I'd prefer a `self.logger.warning` kind of message here. We use logging elsewhere for messages directed to end users, and when the message is about the data rather than the code itself.